src/components/menus/TertiaryNav/TertiaryNav.sass

```diff
@@ -2,6 +2,7 @@
 
 .TertiaryNavContainer
 	display: flex
+	overflow-y: auto
```

Comment:
Love the simplicity! My only word of caution has to do with things like tooltips that'll be clipped if they flow outside the bounds of this container -- even if set to `position: absolute`. That's really a larger issue where we probably need to move to a new tooltip component that uses React [portals](https://reactjs.org/docs/portals.html).